### DISCUSSION TITLE
Support for nested pipeline templates?

### DISCUSSION THREAD

**Discussion by RiviqVH** [go-vela/community#670] [Q&A]
Hi all,

Are nested pipeline templates supported? I can't find anything about nesting templates in the docs, neither did my testing work so far.

If it is supported, I will paste my tests here to find out what I'm doing wrong.

Thanks in advance!
Vladimir

**Comment by wass3r** [comment]
thanks for the question. i know there is some demand for this and we did open an issue for this here: https://github.com/go-vela/community/issues/256. let me know if what you are asking for is different than that though or feel free to upvote it. unfortunately, at this time it has not been prioritized yet for any upcoming sprint.

**Reply by RiviqVH** [reply]
@wass3r Thanks for the reply. That is exactly what I was referring to, thanks for linking the issue. I've upvoted it and hope to see it implemented in the future.
